microcosm_flask/swagger/parameters/decorated.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from marshmallow.fields import Field
+
+from microcosm_flask.swagger.parameters.base import ParameterBuilder
+
+
+SWAGGER_TYPE = "__swagger_type__"
+SWAGGER_FORMAT = "__swagger_format__"
+
+
+class DecoratedParameterBuilder(ParameterBuilder):
```

Comment:
One of the motivations for this PR is the ability to extend the decorated builder to do more things. The surface area for such changes is now small enough that this is sane.